.ci/install_crio.sh

```diff
@@ -38,7 +39,11 @@ popd
 
 echo "Installing CRI-O"
 make clean
-make
+if [ "$ID" == "centos" ] || [ "$ID" == "fedora" ]; then
+	make BUILDTAGS='exclude_graphdriver_devicemapper libdm_no_deferred_remove'
```

Comment:
`exclude_graphdriver_devicemapper` what does this mean in details ? I am asking because I am curious to understand how this got fixed.